src/backend/joanie/core/api/client/__init__.py

```diff
@@ -1513,7 +1486,7 @@ class NestedOrderCourseViewSet(NestedGenericViewSet, mixins.ListModelMixin):
     filterset_class = filters.NestedOrderCourseViewSetFilter
     ordering = ["-created_on"]
     queryset = (
-        models.Order.objects.filter(state=enums.ORDER_STATE_VALIDATED)
+        models.Order.objects.filter(state=enums.ORDER_STATE_COMPLETED)
```

Comment:
We should include all states that allow the user to start the training so `ORDER_STATE_ALLOW_ENROLLMENT` according to my previous comment.

Reply:
TODO added.